raaghu-elements/src/rds-tab-group/rds-tab-group.stories.tsx

```
@@ -5,6 +5,21 @@ import RdsTabGroup, { TabType, TabState } from "./rds-tab-group";
   title: "Elements/Tab",
   component: RdsTabGroup,
+  parameters: { 
+    docs: {
+      source: {
+        transform: (code: string) => {
+          // Transform type enum - remove spaces and transform
+          code = code.replace(/type="([^"]+)"/g, (match, p1) => `type={TabType.${p1.replace(/\s+/g, "")}}`);
+          code = code.replace(/type:\s*"([^"]+)"/g, (match, p1) => `type:TabType${p1.replace(/\s+/g, "")}`);
+          //Transform state enum - remove spaces and transform
+          code = code.replace(/state="([^"]+)"/g, (match, p1) => `state={TabState.${p1.replace(/\s+/g, "")}}`);
+          code = code.replace(/state:\s*"([^"]+)"/g, (match, p1) => `state:TabState${p1.replace(/\s+/g, "")}`);
```

ESLint

[error] Expected indentation of 20 spaces but found 10.

(indent)